styles.css

```diff
@@ -0,0 +1,119 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap');
+
+html {
+  display: block;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+.header {
+  background: #3c3a39;
+  display: flex;
+}
+
+.nav-bar {
+  height: 100%;
+  width: 100%;
+  display: flex;
+  align-items: baseline;
+}
+
+.name a {
+  text-decoration: none;
+  padding: 10px;
```

Comment:
[Optional]

- [ ] Please consider updating your padding in the header (both logo and hamburger button) to make it match the Figma design (in Figma it's 24px)

## Your design:
![image](https://user-images.githubusercontent.com/50721479/185491021-81793680-59c8-4105-8476-116cac5bd672.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/185491121-4e3e3090-74cf-4053-8498-12dc91e28f2a.png)